### PR TITLE
create default locale and allow empty name for a locale

### PR DESCRIPTION
**Pull request by petersg83**

### CODE REVIEW

File: packages/strapi-plugin-i18n/constants/index.js [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 const isoLocales = require('./iso-locales');
 
+const DEFAULT_LOCALE = {
+  name: 'English',
+  code: 'en',
```
Comment (alexandrebodin) [inline packages/strapi-plugin-i18n/constants/index.js:7]: is en a valid code ? I think we should peak the en-US from iso-locales instead

Comment (petersg83) [reply packages/strapi-plugin-i18n/constants/index.js:7]: Yes it is :)
It's the locale specified in the US. May need to ask to @sam-pires 
Why do you prefer "en-US"?

Comment (sam-pires) [reply packages/strapi-plugin-i18n/constants/index.js:7]: That's the way it has been specified back in the days. 
I don't have all the why behind that decision but from what I saw during the exploration phase in most cases users will use 1 locale by language(en, fr, it), and only a few numbers of interviewees have this specific level of localization(en-US, en-CA).
"en" seems to better fit our users' use cases than "en-US"

Comment (alexandrebodin) [reply packages/strapi-plugin-i18n/constants/index.js:7]: If en exists as a single locale I'm good 👍  jsut thought it wasn't valid :)

**Review by alexandrebodin — APPROVED** [review]

**Pull request merged**